publisher.rb

```diff
@@ -0,0 +1,58 @@
+require 'aws-sdk-sqs'
+class Publisher
+  # Note: You can store the values of the aws access keys and region list in a config file (json) 
+  # from where you can get the values into the constants.
+  ACCESS_KEY_ID = #<your access key id>
+  SECRET_ACCESS_KEY = #<your secret access key>
+  REGION = #<your aws account region>
+
+  attr_accessor :sqs
+
+  def initialize
+    creds = Aws::Credentials.new(ACCESS_KEY_ID, SECRET_ACCESS_KEY)
+    @sqs = Aws::SQS::Client.new(region: REGION, credentials: creds)
+  end
+
+  # For Standard Queue
+  def create_std_q queue_name
```

Comment:
Use parenthesis  when passing arguments to a method. You can leave it in case a method does not accept any arguments.
For L17, L22, L27, L42